web/client-api/client-api.gradle

```
@@ -159,6 +160,37 @@ def gwtIntegrationTest = tasks.register('gwtIntegrationTest', Test) { t ->
     t.scanForTestClasses = false
 }
 
+tasks.register('manualGwtTest', Test) {t ->
+    t.description = '''Test wiring to run either unit or integration tests with a manual browser and an already-running server.
+This makes it easier to run a tests repeatedly, either one at a time or as a class/suite, without
+paying to start/stop selenium and deephaven each time. The port will remain constant at 8888 each
+run to let breakpoints continue to work across repeated runs.
+
+To use this, first start a server on port 10000 with anonymous access enabled. Then, either select
+a test in IntelliJ to run using the manualGwtTest task, or invoke from the command line with info
+logging enabled and a specific test selected, e.g.:
+./gradlew :web-client-api:manualGwtTest --info --tests io.deephaven.web.client.api.NullValueTestGwt
+
+When the URL appears to run in your browser, click on it, or refresh an existing browser window.'''
```

Kind of confused when I first read this, suggest modifying the wording a bit.
```suggestion
Click the URL that is printed out to run the test in your browser, or refresh an existing browser window.'''
```